src/components/admin/form-components/withValidate.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { getDisplayName } from 'lib/higher-order-helpers';
+
+const withValidate = (...validators) => (WrappedField) => {
+  const ValidatedField = (originalProps) => {
+    const failedValidator = validators
```

Comment:
Intuitively I would think that validators return `true` upon a correct validation and not as it seems to be the case here that `false` means that no errors were found. Could you add a small comment elaborating that =)?

Reply:
I think it's because the validators return a string if there's an error and null if not. I think we could make this clearer by 

1. Add a JSDoc comment above the function 
```js
/**
 * Write explanation here
 * @param {func} validators - array of validators of the form...
 */
```

2. make the check below `validator(value) !== null`

Reply:
Yeah, @emilgoldsmith nailed it. I'll at the JSdoc comment since it's counter-intuitive. Im up for thinking about my `validator` form, but I think this is actually the cleanest way to handle it as long as I document well.